src/pages/messages/TasksPage.tsx

```diff
@@ -518,18 +267,10 @@ const TasksPage = () => {
           />
 
           <PrivateRoute exact path={UrlPathTemplate.TasksNvi} isAuthorized={isNviCurator}>
-            <NviCandidatesList
-              nviCandidatesQuery={nviCandidatesQuery}
-              nviQueryParams={listNviCandidatesParams}
-              rowsPerPage={rowsPerPage}
-              setRowsPerPage={setRowsPerPage}
-              page={page}
-              setPage={setPage}
-              helmetTitle={t('common.nvi')}
-            />
+            <NviCandidatesList />
           </PrivateRoute>
           <PrivateRoute exact path={UrlPathTemplate.TasksNviStatus} isAuthorized={isNviCurator}>
-            <NviStatusPage activeYear={nviYearFilter} />
+            <NviStatusPage />
```

Comment:
TODO: burde ta vare på query param her, sånn at riktig år også vises for status. Har egen oppgave for å vise år-filter, men naturlig å fikse visnign av riktig år her med en gang tenker jeg